config/query-api.yml

```diff
@@ -66,6 +66,28 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  /v1/webhook:
+    post:
+      summary: Data to add to RDS
+      description: Add data
```

Comment:
I'd add a little detail to the summary and description here.

Reply:
👍